modules/@apostrophecms/login/index.js

```diff
@@ -154,6 +154,47 @@ module.exports = {
           }
           return requirement.props(req, user);
         },
+        async requirementVerify(req) {
+          const name = self.apos.launder.string(req.body.name);
+
+          const { user } = await self.findIncompleteTokenAndUser(req, req.body.incompleteToken);
+
+          const requirement = self.requirements[name];
+
+          if (!requirement) {
+            throw self.apos.error('notfound');
+          }
+
+          if (!requirement.verify) {
+            return {};
```

Comment:
Should throw an error. A requirement with no verify function is broken.

Reply:
Done